.config/nextest.toml

```diff
@@ -0,0 +1,6 @@
+[profile.ci]
+retries = 3
+status-level = "all"
```

Comment:
could you please document these options?

Reply:
Would placing a link to the official docs above `[profile.ci]` do the job, too?

Reply:
ok